sprokit/src/sprokit/pipeline_util/lex_processor.cxx

```diff
@@ -637,7 +639,7 @@ lex_processor::priv::
 current_loc() const
 {
   // Get current location from the include file stack top element
-  return kwiver::vital::source_location( m_include_stack.back()->m_filename,
+  return kwiver::vital::source_location( std::make_shared< std::string >( *(m_include_stack.back()->m_filename) ),
```

Comment:
Let's leave as is then to simplify the hotfix, since David H will likely revert this patch on his branch